easybuild/easyconfigs/g/GROMACS/GROMACS-2021.4-foss-2021a-CUDA-11.3.1-colvars.eb

```diff
@@ -0,0 +1,82 @@
+name = 'GROMACS'
+version = '2021.4'
+_cuda_suffix = '-CUDA-%(cudaver)s'
+versionsuffix = _cuda_suffix + '-colvars'
+
+homepage = 'https://colvars.github.io/'
+description = """
+GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the
+Newtonian equations of motion for systems with hundreds to millions of
+particles.
+
+This is a GPU enabled build, containing both MPI and threadMPI builds.
+
+It also contains the gmxapi extension for the single precision MPI build.
+
+Colvars is a software module for molecular simulation and analysis that
+provides a high-performance implementation of sampling algorithms defined on a
+reduced space of continuously differentiable functions (aka collective
+variables).
+"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = [
+    'https://github.com/Colvars/gromacs/archive/',
+]
+sources = ['v%(version)s-colvars.tar.gz']
```

Comment:
that filename doesn't seem to be available there (https://github.com/Colvars/gromacs/tags). it is without `-colvars`, but it has a different checksum...

otoh, `v2021.5-colvar` was added 19h ago :)